### ISSUE TITLE
menyoo wont work

### ISSUE THREAD

**Issue by isaacparagon** [MAFINS/MenyooSP#509]
I tried for 6 hours and nothing worked. I watched over 10 different videos on it and still not working I don't know why

**Comment by Acalann** [comment]
What isn't working, tell me exactly what you are doing

**Comment by isaacparagon** [comment]
it wont pull up the menu and F8 will not pull it up i downloaded script hook v

**Comment by vbelevcev** [comment]
script hook ещё не обновили

**Comment by metoxys** [comment]
@isaacparagon A new ScriptHook update has been released today  
Close this issue if that works

**Comment by perccobain** [comment]
Still dosen't work with the new update. Wont even open

> @isaacparagon A new ScriptHook update has been released today Close this issue if that works

**Comment by vbelevcev** [comment]
Why does it work for others?

**Comment by vbelevcev** [comment]
> Still dosen't work with the new update. Wont even open
> 
> > @isaacparagon A new ScriptHook update has been released today Close this issue if that works
Use an alternative version of the download
https://gta.2take1.menu/dev/mdsoab/
and no need to create the same topics, the menu itself is loaded and working

**Comment by nomakewan** [comment]
@perccobain Could you please post ScriptHookV.log from your GTAV folder?

I have the latest GTAV (2845) and the latest release of Menyoo (1.8.1) and Menyoo loads up just fine with zero issues. In fact, Menyoo has never failed to load for me on any version of GTAV. Perhaps by inspecting your log we might see something.

**Comment by DymosCj** [comment]
> u
[ScriptHookV.log](https://github.com/MAFINS/MenyooSP/files/10719844/ScriptHookV.log)

**Comment by DymosCj** [comment]
It hasn't worked for me after the update as well, maybe the only difference is im on steam version?

**Comment by nomakewan** [comment]
I am on Steam as well.

@DymosCj According to your log, you have not yet updated your ScriptHookV. Please update your copy of ScriptHookV to 2824.

**Comment by DymosCj** [comment]
still doesnt work and the scripthook.txt log wont pop back up meaning it is not loading it at all?

**Comment by nomakewan** [comment]
> still doesnt work and the scripthook.txt log wont pop back up meaning it is not loading it at all?
This means there's probably a permissions issue with your GTAV folder, and so ScriptHookV doesn't have the ability to write its logs (and probably won't have proper permissions to load mods and trainers either). Did you by chance have Steam's default installation folder in Program Files? I've seen a lot of people have issues with mods when their Steam Library is Program Files.

My Steam library is on a separate drive entirely. It is strange however that you managed to write a log back in 2802 but not in 2824.

**Comment by DymosCj** [comment]
i agree, ill look around to see if i can figure it out, the anti viruses are off, both windows and windows defualt, i do have my steam files on a external hard drive as well,

**Comment by nomakewan** [comment]
So it sounds like everyone who thinks they have a Menyoo issue are actually having an issue with SHV not loading correctly. If SHV isn't generating a log on start, then it's not loading. If it's not loading, then nothing is going to work, including of course trainers like Menyoo.

**Comment by DymosCj** [comment]
> So it sounds like everyone who thinks they have a Menyoo issue are actually having an issue with SHV not loading correctly. If SHV isn't generating a log on start, then it's not loading. If it's not loading, then nothing is going to work, including of course trainers like Menyoo.

Sounds bout right, how can we get this fixed lol

**Comment by DymosCj** [comment]
Was looking through to see what i could do and found this in my directory
[asiloader.log](https://github.com/MAFINS/MenyooSP/files/10729884/asiloader.log)

**Comment by DymosCj** [comment]
seems to be using a dinput8.dll from systems 32, is this normal?

**Comment by DymosCj** [comment]
Fixxed it by downloading the “Menyoo Nightly menu” and downloading scripthook again, manually deleting scripthook.dll, menyoo.asi, and menyooStuff folder and replacing with these files 

https://www.dev-c.com/gtav/scripthookv/

https://github.com/MAFINS/MenyooSP

![CEFC20C4-9A78-45C8-8A68-59C3769650FB](https://user-images.githubusercontent.com/125252002/218673565-f38e1ff3-3a07-4157-963c-f5fab6a0f363.jpeg)


I also went into the ASILOADER And changed the asi loader text 

LIB: “YOUR DINPUT.DLL LOCATION And saved it! 


PLEASE LET ME KNOW IF THIS FIXXES YOUR GAME, IT FIXXED MINE!

**Comment by nomakewan** [comment]
>seems to be using a dinput8.dll from systems 32, is this normal?

It isn't. ScriptHookV comes with dinput8.dll which is a modified version that is what allows SHV to work in the first place. That modified version of dinput8.dll is the ASI Loader. If you remove this file, or replace it with the one from system32, then nothing will work.

As a note renaming/moving/removing dinput8.dll is the easy way to 'unmod' your game in case you want to go online, too.

**Comment by DymosCj** [comment]
> > seems to be using a dinput8.dll from systems 32, is this normal?
> 
> It isn't. ScriptHookV comes with dinput8.dll which is a modified version that is what allows SHV to work in the first place. That modified version of dinput8.dll is the ASI Loader. If you remove this file from your GTAV folder, or replace it with the one from system32, then nothing will work.
> 
> As a note renaming/moving/removing dinput8.dll is the easy way to 'unmod' your game in case you want to go online, too.
> 
> Also editing the asiloader.log file does nothing at all. It's a log, not a config.

Besides the point I fixxed it look up

**Comment by nomakewan** [comment]
What likely fixed it was reinstalling ScriptHookV and dinput8.dll. Editing asiloader.log literally does nothing, and you likely didn't need the nightly build of Menyoo either. But hey, glad it's working for you now.
